resources/views/projects/form.blade.php

```diff
@@ -33,13 +34,14 @@
         <div class="form-group">
             <label class="form-label required" for="total_cost">Total Cost</label>
             <div class="form-control-wrap">
-                <input type="text" class="form-control" id="total_cost" value="@isset($project){{ $project->total_cost }}@endisset" name="total_cost" required>
+                <input type="number" class="form-control" id="total_cost" value="@isset($project){{ $project->total_cost }}@endisset" name="total_cost" required>
```

Comment:
what about if i will type values in decimal ?

Reply:
In this condition it will not accept decimal, but after adding step="any" attribute it will also accept decimals.